engine/src/main/java/org/terasology/math/ChunkMath.java

```diff
@@ -64,35 +67,188 @@ public static int calcChunkPosZ(int z, int chunkPowerZ) {
     public static int calcChunkPosX(int x) {
         return calcChunkPosX(x, ChunkConstants.CHUNK_POWER.x);
     }
+
     public static int calcChunkPosY(int y) {
         return calcChunkPosY(y, ChunkConstants.CHUNK_POWER.y);
     }
+
     public static int calcChunkPosZ(int z) {
         return calcChunkPosZ(z, ChunkConstants.CHUNK_POWER.z);
     }
 
+    /**
+     *
```

Comment:
Bonus for dropping in even a brief line at the start of the javadoc just to cover the method overall 👍